src/sql/sqlops.proposed.d.ts

```diff
@@ -203,4 +203,116 @@ declare module 'sqlops' {
 		 */
 		export function registerModelViewProvider(widgetId: string, handler: (view: ModelView) => void): void;
 	}
+
+	export namespace window {
+		export namespace modelviewdialog {
```

Comment:
we should not name the interface modal if we might change it to be not modal in future